integreat-compass-cli.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+"""Django's command-line utility for administrative tasks."""
+import os
+import sys
+
+
+def main():
+    """Run administrative tasks."""
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "integreat_compass.settings")
+
+    # Override default port for `runserver` command
+    from django.core.management.commands.runserver import Command as runserver
```

Comment:
Is there anything that speaks against moving this up to the regular imports?

Reply:
No, will do.